test/cli.js

```diff
@@ -188,5 +188,36 @@ describe('Belly Button CLI', function () {
         done();
       });
     });
+
+    it('defaults to belly-button style', function (done) {
+      var lintFile = Path.join(__dirname, 'fixtures', 'config', 'yoda.js');
```

Comment:
Can you use `fixturesDirectory` (defined at the top of the file) instead of `__dirname` and `'fixtures'`.